modules/openapi-generator/src/main/resources/Java/libraries/webclient/build.gradle.mustache

```diff
@@ -78,8 +84,14 @@ if(hasProperty('target') && target == 'android') {
     apply plugin: 'java'
     apply plugin: 'maven-publish'
 
+    {{#useJakartaEe}}
+    sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
why 11?

Reply:
Was a mistake. I forgot to push my latest changes. Is now up 2 date. Thanks!